### PR TITLE
Add meta to Security documents (User, Profile, Role)

### PR DESCRIPTION
**Pull request by samniisan**
## Introduces
- Meta in Security documents

## Boyscoot
- Removed `getPaginatedIdsFromQuery` function from `elasticsearch.js` service in favor of `getAllIdsFromQuery`
- Typos

## Linked PRs
- Backoffice: https://github.com/kuzzleio/kuzzle-backoffice/pull/327
- SDK JS: https://github.com/kuzzleio/sdk-javascript/pull/236
- SDK PHP: https://github.com/kuzzleio/sdk-php/pull/81
- SDK Android: https://github.com/kuzzleio/sdk-android/pull/144
- Documentation: https://github.com/kuzzleio/documentation/pull/320

### CODE REVIEW

**Comment by kuzzle** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [index.js#L89](https://github.com/kuzzleio/kuzzle/blob/80b9ab57aeb04cc17c6cd3fb730bb3ed5bcd15b8/lib/services/internalEngine/index.js#L89): Complete the task associated to this TODO comment. [rule](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'

**Comment by kuzzle** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [index.js#L89](https://github.com/kuzzleio/kuzzle/blob/64727ceb80f55fb15cd0558c879785b9576421e8/lib/services/internalEngine/index.js#L89): Complete the task associated to this TODO comment. [rule](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'

File: lib/api/core/auth/formatProcessing.js [hunk 1]
```diff
@@ -31,12 +31,14 @@ module.exports = {
   formatRoleForSerialization: role => {
     const response = {_id: role._id};
 
-    response._source = _.assignIn({}, role);
+    response._source = _.assignIn({}, _.omit(role, ['_kuzzle_info']));
```
Comment (scottinet) [inline lib/api/core/auth/formatProcessing.js]: `_.omit` already returns a copy of the provided object (without the properties to be omitted), so using `_.assignIn` on an empty object to create another copy is superfluous

File: lib/api/core/auth/formatProcessing.js [hunk 2]
```diff
@@ -49,9 +51,11 @@ module.exports = {
   formatProfileForSerialization: profile => {
     const response = {_id: profile._id};
 
-    response._source = _.assignIn({}, profile);
+    response._source = _.assignIn({}, _.omit(profile, ['_kuzzle_info']));
```
Comment (scottinet) [inline lib/api/core/auth/formatProcessing.js]: ditto

File: lib/api/core/auth/formatProcessing.js (hunk 1 above)
Comment (samniisan) [inline lib/api/core/auth/formatProcessing.js]: Indeed, done

**Comment by kuzzle** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [index.js#L89](https://github.com/kuzzleio/kuzzle/blob/754e86e7b01fb20be93cd50b9c4099ac42bb76f3/lib/services/internalEngine/index.js#L89): Complete the task associated to this TODO comment. [rule](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'

**Pull request merged**